spec/features/01_user_relations_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+feature "user visits their show page" do
+  let!(:user_1) { User.create(email: 'jscho13@gmail.com',
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.